app/services/course/survey/reminder_service.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+class Course::Survey::ReminderService
+  include Course::ReminderServiceConcern
+
+  class << self
+    delegate :opening_reminder, to: :new
+    delegate :closing_reminder, to: :new
+  end
+
+  def opening_reminder(survey, token)
+    return unless survey.opening_reminder_token == token && survey.published?
```

Comment:
Should we also just return if `survey.start_at` has passed? Otherwise we have reminder jobs that would never run? @allenwq